src/components/SummarySection.jsx

```diff
@@ -0,0 +1,60 @@
+import PropTypes from "prop-types";
+import Go123Box from "./Go123Box";
+import SummaryBox from "./SummaryBox";
+
+const SummarySection = ({
+  className,
+  gridClassName,
+  bgColor,
+  textColor,
+  has123Go,
+  goBgColor,
+  goTextColor,
+  goDescription,
+  summaryList,
+}) => {
+  return (
+    <div className={className}>
+      <div className={`grid ${gridClassName}`}>
+        {has123Go && (
+          <Go123Box
+            className={className}
```

Comment:
don't think you want these classNames to get propagated to the sub-components, remove this